velox/docs/develop/operators.rst

```diff
@@ -232,13 +232,77 @@ followed by the group ID column. The type of group ID column is BIGINT.
      - Description
    * - groupingSets
      - List of grouping key sets. Keys within each set must be unique, but keys can repeat across the sets.
+     - Grouping keys are specified with their output names.
    * - groupingKeyInfos
      - The names and order of the grouping key columns in the output.
    * - aggregationInputs
      - Input columns to duplicate.
    * - groupIdName
      - The name for the group-id column that identifies the grouping set. Zero-based integer corresponding to the position of the grouping set in the 'groupingSets' list.
 
+To illustrate why GroupingSets should use output column names lets examine the following SQL query:
+
+.. code-block:: sql
+
+  select COUNT(orderkey), count(distinct orderkey) from orders;
+
+In this query the user wants to compute aggregates on the same key, though with
+and without the DISTINCT clause. With a particular optimization strategy
+optimize.mixed-distinct-aggregations, Presto uses GroupIdNode to compute these.
```

Comment:
Can we link here to Presto's documentation about this property?

Reply:
I just found https://prestodb.io/docs/current/release/release-0.156.html?highlight=mixed%20distinct%20aggregations. Do you want to link this ?

Reply:
I found this post which seems useful to link to: https://www.qubole.com/blog/presto-optimizes-aggregations-over-distinct-values

Reply:
Also this seems useful: https://github.com/trinodb/trino/pull/15927

Reply:
Thanks for the link. Have added it.